travis_configure.sh

```diff
@@ -35,6 +35,11 @@ before_install()
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
             gem install fpm
+            if [[ -n "$TRAVIS_TAG" ]]; then
```

Comment:
Well, jfrog is not needed to install anything, it's just needed to automatically upload files to bintray. If you build locally, even if you build the package, you don't need `jfrog`.